src/highlight/HighlightAnnotation.scss

```diff
@@ -0,0 +1,11 @@
+.ba-HighlightAnnotation:focus {
+    outline: none;
+
+    .ba-HighlightAnnotation-rect {
+        fill: rgba(255, 235, 127, 1);
```

Comment:
Nit: is this color reused somewhere else? Maybe we can have this moved to a constants file where it's shared by different places that are using it.